lib/puppet/ssl/validator/default_validator.rb

```diff
@@ -130,13 +126,33 @@ def setup_connection(connection)
     end
   end
 
+  ##
+  # Decode a string of concatenated certificates
+  #
+  # @return [Array<OpenSSL::X509::Certificate>]
+  def decode_cert_bundle(bundle_str)
```

Comment:
I've implemented something very similar to this in my PR, for use in the host class when loading certs from disk. It might be useful to refactor this into some sort of utility module so both places can use it.

Reply:
I've also added something similar onto the Host class for loading CRLs. Should we glom all these in in separate PRs and then try to refactor. Or should one of us create a module that other's then depend on for their work?

Reply:
This is stolen/absorbed from the removal of the `Puppet::SSL::Configuration` class. Does it make sense to try and keep this method clean in its current form, or should we do a rewrite of it from either of your PRs?

Reply:
Either way works for me. I just ran into needing this in a test though, so as long as we wind up with this in some central location, I'm happy.

Reply:
I think it might be cleanest to have two methods in a utility module, one for certs and one for crls, maybe that call down into one common implementation with the different dividers supplied.

Reply:
I am a fan of glomming in and then refactor as needed.